broker-cli/commands/wallet.js

```diff
@@ -201,13 +202,83 @@ async function networkAddress (args, opts, logger) {
+      colWidths: [14, 24, 24],
+      style: { head: ['gray'] }
+    })
+
+    statusTable.push(['Active', '', ''])
+    statusTable.push([`  Buy ${baseSymbolCapacities.symbol.toUpperCase()}`, formatBalance(baseSymbolCapacities.activeReceiveCapacity, 'active'), formatBalance(counterSymbolCapacities.activeSendCapacity, 'active')])
+    statusTable.push([`  Sell ${baseSymbolCapacities.symbol.toUpperCase()}`, formatBalance(baseSymbolCapacities.activeSendCapacity, 'active'), formatBalance(counterSymbolCapacities.activeReceiveCapacity, 'active')])
+
+    statusTable.push(['Pending', '', ''])
+    statusTable.push([`  Buy ${baseSymbolCapacities.symbol.toUpperCase()}`, formatBalance(baseSymbolCapacities.pendingReceiveCapacity, 'pending'), formatBalance(counterSymbolCapacities.pendingSendCapacity, 'pending')])
+    statusTable.push([`  Sell ${baseSymbolCapacities.symbol.toUpperCase()}`, formatBalance(baseSymbolCapacities.pendingSendCapacity, 'pending'), formatBalance(counterSymbolCapacities.pendingReceiveCapacity, 'pending')])
+
+    statusTable.push(['Inactive', '', ''])
+    statusTable.push([`  Buy ${baseSymbolCapacities.symbol.toUpperCase()}`, formatBalance(baseSymbolCapacities.inactiveReceiveCapacity, 'inactive'), formatBalance(counterSymbolCapacities.inactiveSendCapacity, 'inactive')])
+    statusTable.push([`  Sell ${baseSymbolCapacities.symbol.toUpperCase()}`, formatBalance(baseSymbolCapacities.inactiveSendCapacity, 'inactive'), formatBalance(counterSymbolCapacities.inactiveReceiveCapacity, 'inactive')])
+
+    logger.info(` ${market.bold.white}`)
+    logger.info(statusTable.toString())
+  } catch (e) {
+    logger.error(e)
+  }
+}
+
+/**
+ * Formats the capacities to fixed numbers and colors them if they are greater
+ * than 0
+ *
+ * @function
+ * @param {string} balance
+ * @param {string} status
+ * @return {string} balance formatted with size and color
+ */
+function formatBalance (balance, status) {
+  const fixedBalance = Big(balance).toFixed(16)
+  if (Big(balance).gt(0)) {
+    switch (status) {
+      case 'active':
+        return fixedBalance.green
+      case 'pending':
+        return fixedBalance.yellow
+      case 'inactive':
+        return fixedBalance.red
+    }
+  } else {
+    return fixedBalance
+  }
+}
+
 module.exports = (program) => {
   program
     .command('wallet', 'Commands to handle a wallet instance')
-    .help('Available Commands: balance, new-deposit-address, commit-balance, network-address')
+    .help('Available Commands: balance, new-deposit-address, commit-balance, network-address, network-status')
     .argument('<command>', '', Object.values(SUPPORTED_COMMANDS), null, true)
     .argument('[sub-arguments...]')
     .option('--rpc-address', 'Location of the RPC server to use.', validations.isHost)
+    .option('--market [marketName]', 'Relevant market name', validations.isMarketName)
```

Comment:
don't you want to put this below network-status?

Reply:
The options only work if they are above the action.

Reply:
@treygriffith because of the issues w/ sub-commands, we have to put it in both places unfortunately